react/components/Table/Toolbar.js

```diff
@@ -270,17 +272,20 @@ class Toolbar extends PureComponent {
             <div title={download.label} className="mh2">
               <ButtonWithIcon
                 icon={
-                  <span className="c-on-base mh2">
+                  <span
+                    className={`${download.disabled ? '' : forcedColor} mh2`}>
                     <IconDownload size={MEDIUM_ICON_SIZE} />
                   </span>
                 }
-                disabled={loading}
                 variation="tertiary"
+                disabled={download.disabled}
                 isLoading={download.isLoading}
                 size="small"
                 onClick={download.handleCallback}>
                 {download.label && (
-                  <span className="c-on-base">{download.label}</span>
+                  <span className={`${download.disabled ? '' : forcedColor}`}>
```

Comment:
Why do you need to force the color here? Wouldn’t the disable in the `ButtonWithIcon` component set this color?

Reply:
It's forced as a design decision. @cmdalbem might clarify this point.

Reply:
LGTM then :sparkles: